security/security-spring/src/main/resources/applicationContext-security.xml

```diff
@@ -88,6 +88,9 @@
     <http pattern="/api/swagger-ui.html" security="none"/>
     <http pattern="/api/health" security="none"/>
     <http pattern="/api/cache/**" security="none"/>
+    <http pattern="/api/cacheStatistics" security="none"/>
+    <http pattern="/api/**/keysInCache" security="none"/>
+    <http pattern="/api/**/keyCountsPerClass" security="none"/>
```

Comment:
@pvannierop @dianab0 i resolved the conflicts with https://github.com/cBioPortal/cbioportal/pull/9046. Does this look good? Do we still need these other ones?

Reply:
@inodb This looks good!